### ISSUE TITLE
Random Client Tick Crash

### ISSUE THREAD

**Issue by WenXin20** [Cadiboo/NoCubes#53]
Minecraft v1.12.2
Forge v14.23.2838
No Cubes v0.3.3

I will randomly get this crash. I'm not sure what causes it or when it will happen, such as in this case I was only trying to craft. Terrain collisions, Leaf Smoothing, and Terrain Smoothing were all disabled at the time.
[Crash Report](https://paste.dimdev.org/uvijeqirun.mccrash)

**Comment by Cadiboo** [comment]
Does this happen in multiplayer? Can you give more details please?
This line shouldn't really ever crash
https://github.com/Cadiboo/NoCubes/blob/f66cd0bb87f41381d5da875429a3414cbd95f9d2/src/main/java/io/github/cadiboo/nocubes/client/ClientEventSubscriber.java#L75

**Comment by WenXin20** [comment]
It's in singleplayer. It just seems to randomly  happen and I don't know how to reproduce it. Sometimes my game would be fine for several days before it happens again. 

But I have noticed that it seems more likely to happen if another mod has an error or crash as well. 
For example, before it got fixed, LittleTiles would have a crash whenever I tried exciting the game, which seemed to cause NoCubes to crash right afterwards too. I also have VanillaFix installed. 
But then again, it sometimes crashes all on its own. Kinda odd.
